members_and_sponsors.md

```diff
@@ -42,3 +41,6 @@ Members with affiliations:
 - Ehsan Totoni - Bodo
 - Hadia Ahmed - Bodo
 - Emaan Hariri - Bodo
+- Matthew Roeschke - NVIDIA, Pandas
+- Sebastian Berg - NumPy
```

Comment:
Thanks for asking. Affiliations of employer are indeed tied to funding. The rule we used for affiliations was:
- Any array or dataframe libraries you maintain (not dependent libraries like scikit-learn, scipy, etc.)
- If employed by a consortium sponsor, then list that (only exception is that we gave NVIDIA a bit of wiggle room because they spent significant engineering time, and sponsorship discussions are in progress).